table/server/common/src/main/java/alluxio/table/common/transform/TransformPlan.java

```diff
@@ -44,23 +42,29 @@
    * @param definition the transformation definition
    */
   public TransformPlan(Layout baseLayout, Layout transformedLayout,
-      TransformDefinition definition) {
+                       TransformDefinition definition) {
     mBaseLayout = baseLayout;
     mTransformedLayout = transformedLayout;
     mJobConfigs = computeJobConfigs(definition);
   }
 
   private ArrayList<JobConfig> computeJobConfigs(TransformDefinition definition) {
-    final List<TransformAction> actions = definition.getActions();
+    ArrayList<JobConfig> actions = new ArrayList<>();
+    Layout baseLayout = mBaseLayout;
+    boolean deleteSrc = false;
```

Comment:
What is this for?

Reply:
It's for the case when transformations need to be chained and therefore, for every transformation after the first, you need to deleteSrc.